src/components/variantsListPanel/VariantsPanelList.vue

```diff
@@ -41,6 +41,23 @@ export default {
             this.$store.commit('setListMounting', false);
         }
     },
+    updated() {
+        const ItemElement = document.querySelector('.variants-list > .variants-list_item__active');
+        const ListElement = document.querySelector('.baron__scroller');
```

Comment:
All custom scrollbars use baron__scroller class, so this query can remove another element. 
This works as long as the given element is the first in the DOM.